master/internal/stream/trials.go

```diff
@@ -101,6 +101,15 @@ func getTrialMsgsWithWorkspaceID(trialMsgs []*TrialMsg) *bun.SelectQuery {
 	return q
 }
 
+func permFilter(
```

Comment:
question (non-blocking): what was the motivation for changing from a private function?

Reply:
wanted to use it in experiments, so refactored it to be accessible to experiments

Reply:
Good call, it might be worth moving to a separate file like a `internal/stream/utils.go`

Reply:
good call!